src/its_preselector/controlbyweb_web_relay.py

```diff
@@ -143,6 +160,27 @@ def get_status(self):
                     for relay_state in relay_states:
                         matches = matches and self.state_matches(relay_state, xml_root)
                     state[key] = matches
+
+                if "sensors" in self.config:
+                    for key, value in self.config["sensors"].items():
+                        try:
+                            state[key] = self.get_sensor_value(value)
```

Comment:
A check should be added to ensure that key names are unique- otherwise, there is potential for keys in `state` to be overwritten without error.

Reply:
Another option could be to make `state` a nested dictionary

Reply:
Modified to raise a ConfigurationException if names are duplicated within status_states, sensors or inputs.